gradle.properties

```diff
@@ -1,18 +0,0 @@
-# Project-wide Gradle settings.
```

Comment:
That seems okay. If we need gradle.properties later, we can load in the secrets from another file and add that (like http://stackoverflow.com/questions/21999829/how-do-i-read-properties-defined-in-local-properties-in-build-gradle)